provisioning/roles/supervisor/handlers/main.yml

```diff
@@ -1,5 +1,5 @@
 - name: restart supervisor
-  shell: /etc/init.d/supervisor stop; /etc/init.d/supervisor start
+  shell: /etc/init.d/supervisor stop; /etc/init.d/supervisor start; exit 0
```

Comment:
We've got the issue on PIA. Didn't find a way to avoid it at the time, this works, but Solr won't be started correctly on the first provisioning.